lib_eio/process.ml

```diff
@@ -7,8 +7,8 @@ type status = [ exit_status | `Stopped of int ]
 
 let pp_status ppf = function
   | `Exited i -> Format.fprintf ppf "Exited %i" i
-  | `Signaled i -> Format.fprintf ppf "Signalled %i" i
-  | `Stopped i -> Format.fprintf ppf "Stopped %i" i
+  | `Signaled i -> Format.fprintf ppf "Signalled %a" Fmt.Dump.signal i
```

Comment:
Is it _signaled_ or _signalled_? :)

Reply:
Good point. I guess the single-L is to match Unix's `WIFSIGNALED`. I've changed the wording to avoid the problem.